install/update_0845_0846.php

```diff
@@ -63,13 +63,19 @@ function update0845to0846() {
                                  true);
    }
 
+   // TODO : can be improved once DBmysql->update() supports JOIN
+   $table   = DBmysql::quoteName("glpi_documents_items") . " AS " . DBmysql::quoteName("doc_i");
```

Comment:
You can quote the whole string at once, quote function handles the AS correctly.
```suggestion
    $table   = DBmysql::quoteName("glpi_documents_items AS doc_i");
```

Reply:
I've reverted this change as update with joins are not cleanly supported.